### PR TITLE
added a validation for user input

### PR DESCRIPTION
**Pull request by DerMoehre**

### CODE REVIEW

File: rockpaperscissor.py [hunk 1]
```diff
@@ -27,9 +27,17 @@
 ---.__(___)
 ''')
 
+# set initial value for user_move
+user_move = -1
+
 game_ascii = [Rock, Paper, Scissors]
 
-user_move = int(input("What do you choose for rock 0, for paper 1 for scissors 2.:-"))
+while (user_move < 0) or (user_move > 2):
```
Comment (LastDeadlock) [inline rockpaperscissor.py:35]: if you add if statement here then it will be great

**Review by LastDeadlock — COMMENTED** [review]
ok great but in this one user can't able to interact

**Comment by LastDeadlock** [comment]
thanks but there are more bugs now well will open another issue soon

File: rockpaperscissor.py (hunk 1 above)
Comment (DerMoehre) [reply rockpaperscissor.py:35]: what is the if statement needed for? 
the user will be questioned to insert an input, as long as the input is between 0 and 2. 
With the try except block, we check for a ValueError, which would rise, if the user inserts a float number.
if the user does enter a float, a print statement will show up, to please insert a valid input

**Comment by DerMoehre** [comment]
> thanks but there are more bugs now well will open another issue soon

what is the unwanted behavior?

**Comment by LastDeadlock** [comment]
With the while loop it's going to loop out in the terminal 🙂 if user
enters anything except 0,1 or 2 you try this by yourself by entering the
value 3 and the reason is that the last else statement will not be executed
because the previous elif statements will takes the lead.

**Comment by DerMoehre** [comment]
ah okay, so you want to end the game, if the user enters a wrong number?
With the while loop, the game will just keep asking the user for correct input

**Comment by LastDeadlock** [comment]
Yes, that's right.

**Pull request merged**